### DISCUSSION TITLE
`개발자의 글쓰기`책 관련 질문

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#92] [Q&A] [answered]
[개발자의 글쓰기](https://www.yes24.com/Product/Goods/79378905) 멘토님이라면 이책을 보셨거나 들어보셨을 것 같아서 혹시 추천할 만한 책인가요?

정리를 자주 하다 보니 문장도 조금 구조적으로 쓰고 싶고 관심이 생겨서 질문드립니다..!

이외에도 혹시 글 쓰기 관련해서 책 추천해주실 수 있나요?

**Comment by jongfeel** [comment, marked as answer]
3년 전에 읽었던 책이군요. 읽어보면 좋긴 합니다. 

https://www.notion.so/jongfeel/e810c550e2d947d8a733ba26fa9bb155?pvs=4

이것도 실제 글을 써보고 판단해 보면 될 거에요.
이미 블로그를 쓰고 계시니까 그 글에 대한 회고와 피드백을 진행해 보면 좋습니다.
